assets/js/meta-box.js

```diff
@@ -0,0 +1,33 @@
+/* global jQuery */
+( function( $ ) {
+
+	$( document ).ready( function() {
+
+		$( 'input[name="primer-layout-override"]' ).change( function() {
+
+			if ( 1 == $( this ).val() ) {
```

Comment:
@fjarrett Loose comparison in JS is not recommended. http://i.stack.imgur.com/35MpY.png